### PR TITLE
Enable e2e test to run HO form main with OCP 4.12

### PR DESCRIPTION
**Pull request by enxebre**
Ref https://issues.redhat.com/browse/HOSTEDCP-843

### CODE REVIEW

File: ci-operator/step-registry/hypershift/aws/e2e/nested/4-12/hypershift-aws-e2e-nested-4-12-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    The control plane is hosted on a dedicated standalone OCP cluster running
+    HyperShift in the `osd-hypershift` AWS account and is destroyed after tests
+    complete.
+
+    The HyperShift launch capability is currently supported by the HyperShift
+    team. For now, please direct all questions and comments to:
+
+    - Alberto Lamela (agarcial@redhat.com)
+    - Seth Jennings (sjenning@redhat.com)
+    - Alvaro Aleman (alvaroaleman@redhat.com)
```
Comment (bryan-cox) [inline ci-operator/step-registry/hypershift/aws/e2e/nested/4-12/hypershift-aws-e2e-nested-4-12-workflow.yaml:20]: Should he be removed from this list?

**Comment by sjenning** [comment]
Can we pass the HC release image (latest and previous) in as environment variables so we don't have to have a separate workflow per release version?

**Comment by enxebre** [comment]
closing in favour of github.com/openshift/release/pull/37292

**Pull request closed without merging**